src/sentry/utils/cloudfunctions.py

```diff
@@ -60,7 +60,7 @@ def upload_function_files(client, code, env_variables):
     f = BytesIO()
     with ZipFile(f, "w") as codezip:
         codezip.writestr("function.js", code)
-        codezip.writestr("index.js", WRAPPER_JS)
+        # codezip.writestr("index.js", WRAPPER_JS)
```

Comment:
@vpandiarajan20 did you mean to leave this?

Reply:
Yes, right now we don't need index.js and it actually breaks the cloud function. In the future, we'll need to change the start point of the user defined function and have index.js call this function, with monitoring.

Reply:
@vpandiarajan20 can you just remove the line then?